tests/integration/test_orders_api.py

```diff
@@ -33,6 +33,7 @@
 TEST_URL = 'http://www.MockNotRealURL.com/api/path'
 TEST_BULK_CANCEL_URL = f'{TEST_URL}/bulk/orders/v2/cancel'
 TEST_DOWNLOAD_URL = f'{TEST_URL}/download'
+TEST_DOWNLOAD_ACTUAL_URL = f'{TEST_URL}/download_actual'
```

Comment:
There are probably better names and URLs for this. In the actual API, it's `link.planet.com`, but we shouldn't depend on anything but the location header.